lib/haml_lint/linter/README.md

```diff
@@ -501,6 +502,24 @@ will compile and run:
 
 Thus it's best to stay away from writing code this way.
 
+## NoPlaceholders
+
+Don't use HTML placeholder attributes.
+
+**Bad**
+```haml
+%input{ placeholder: 'Placeholders arent very accessible' }
+```
+
+**Good**
+```haml
+#my-details Placeholders arent very accessible
+%input{ 'aria-describedby': 'my-details' }
+```
```

Comment:
I wasn't really sure what to include here, so I defaulted to the most common alternative. Open to any other suggestions.